docs/api-docs/getting-started/rest-api-authentication.md

```diff
@@ -0,0 +1,304 @@
+# Authenticating BigCommerce's REST APIs
```

Comment:
@aglensmith , do you have any reservations about changing the file name here? I saw you mentioned wanting to keep the git history.

Reply:
I initially thought that the other article was being removed (due to all the red in the diff). If that was the case, it'd be better to keep the filename the same. Since the original file isn't being removed, we'll have the git history there. So, should be fine.